### ISSUE TITLE
Locked Error:  extras.result_codes:  {"transaction":"tx_bad_auth"}  Result XDR:  AAAAAAAAAGT////6AAAAAA== 

### ISSUE THREAD

**Issue by ChristianCryptoCurrencyCoin** [stellar/stellar-protocol#627]
Looks like my "Number of existing signatures" is 0 and that is what is causing my issue. If that was the case wouldn't I be locked out? How in the world do I get back into my main account? This has been the most frustrating deal. I can never find any answers and no one responds to my tickets on any wallets. I hope there is something I can do ASAP.

extras.result_codes:
 {"transaction":"tx_bad_auth"} 
Result XDR:
 AAAAAAAAAGT////6AAAAAA==

**Comment by ChristianCryptoCurrencyCoin** [comment]
It says my Account lock status: Locked
Though I only meant it for the one token created. 
This doesn't mean my who portfolio is locked does it? This is just crazy.

**Comment by ChristianCryptoCurrencyCoin** [comment]
Oh my, I somehow did what this guy did. https://www.reddit.com/r/Stellar/comments/awxufy/deleted_my_accounts_secret_key_as_a_signer_somehow/    I can't afford to lose the money in there, what do I do? They just told them they're SOL.

**Comment by ChristianCryptoCurrencyCoin** [comment]
![Only Response](https://user-images.githubusercontent.com/66155522/83947791-dd8ff980-a7de-11ea-8f00-cfb0fa886e7a.png)

**Comment by ChristianCryptoCurrencyCoin** [comment]
I want to believe there is still a workaround, but I haven't received anything other than a Support Ticket from LOBSTR. I hope someone knows something.

**Comment by tyvdh** [comment]
According to https://www.stellar.org/developers/guides/concepts/multi-sig.html if your available signers are all weighted below the thresholds for operations there is, by design, nothing you or anyone can do to move or interact with that account.

The SDF actually made use of this feature to "burn" ~55,000,000,000 XLM back in November.
https://medium.com/stellar-development-foundation/sdfs-next-steps-9ae1d7886a0f
https://stellar.expert/explorer/public/account/GALAXYVOIDAOPZTDLHILAJQKCVVFMD4IKLXLSZV5YHO7VY74IWZILUTO

This feature is also often used by accounts issuing assets to ensure they can no longer issue assets beyond a set amount, thereby protecting any user of that asset to ensure the value doesn't get deflated by future token creation.

Inadvertently enabling this feature is a huge bummer but it's an essential feature.

**Comment by tyvdh** [comment]
Out of curiosity were you following a tutorial, walkthrough or using an app or interface which lead you to create and submit the transaction which removed the signer for your account?

Maybe we can add or suggest language to help folks in the future to avoid similar issues.

**Comment by ChristianCryptoCurrencyCoin** [comment]
There was a YouTube page. Unfortunately, going directly to the Stellar Laboratory didn't flag any warnings. I thought if there would have been any locking it would have just been to lock the creating token. Never have read anything that stated you would lock your whole account. Unfortunately, it seems like a honey trap to me. Sorry that you didn't know that it could happen, but we put the trap there and put a sign a mile from where you were saying don't go for the honey, or you'll fall in the trap. I did look at the documentation, but nothing stood out that I had to be worried about locking my account. Still haven't found where it says that will be what happens. Just got that from hours of searching forums and of course from the response on Github and Lobstr.

________________________________
